packages/cli/src/AbstractServer.ts

```diff
@@ -191,6 +194,10 @@ export abstract class AbstractServer {
 		}
 	}
 
+	private async setInstanceId() {
```

Comment:
I would have expected `setInstanceId()` to accept the instance ID as an argument, instead of retrieving it from UserSettings.

Reply:
I moved it to the configuration method so it's less confusing.